src/ex1/script.js

```diff
@@ -0,0 +1,213 @@
+let tasks = [];
```

Comment:
better use const if the variable is not going to be mutated. tasks will always be an array, you just going to push and remove tasks from it